pkg/rtc/types/interfaces.go

```diff
@@ -446,7 +446,6 @@ type MediaTrack interface {
 
 	UpdateTrackInfo(ti *livekit.TrackInfo)
 	ToProto() *livekit.TrackInfo
-	Version() utils.TimedVersion
```

Comment:
Unused interface method. Had added to get version, but users get proto and look up field. So, removing it for now.